apps/mobile/src/components/pump/TokenLaunchDetail/index.tsx

```diff
@@ -225,29 +227,36 @@ export const TokenLaunchDetail: React.FC<LaunchCoinProps> = ({
           </View>
         )}
 
-        {launch?.liquidity_raised && launch?.threshold_liquidity &&
+        {launch?.liquidity_raised && launch?.threshold_liquidity && (
           <View>
             <View style={styles.detailRow}>
               <Text style={styles.label}>Raised Progress</Text>
               <Text style={styles.value}>
-                {Number(launch?.liquidity_raised).toLocaleString()} / {Number(launch?.threshold_liquidity).toLocaleString()}
+                {Number(launch?.liquidity_raised).toLocaleString()} /{' '}
+                {Number(launch?.threshold_liquidity).toLocaleString()}
               </Text>
             </View>
             <View style={styles.progressBarContainer}>
               <View
                 style={[
-                  styles.progressBarFill,
+                  Number(launch.liquidity_raised) <= Number(launch.threshold_liquidity) - 0.2
```

Comment:
Please add a slippage percentage of 2 percent not a brut 0.2 slippage based on the 10 used now, because we launch between 98 to 100 percent of the threshold liquidity